deltachat-ffi/deltachat.h

```diff
@@ -2507,6 +2507,7 @@ void            dc_stop_ongoing_process      (dc_context_t* context);
 #define         DC_QR_BACKUP                 251
 #define         DC_QR_BACKUP2                252
 #define         DC_QR_WEBRTC_INSTANCE        260 // text1=domain, text2=instance pattern
+#define         DC_QR_SOCKS5_PROXY           270 // text1=host, text2=port
```

Comment:
sometimes there is also an user and password, which is already supported by core manual configuration, would be nice if also via link would work

Reply:
what are the exact parameters for `t.me` then? is there a real-world example in some list?

Reply:
here is a shared link of a proxy configuration when you enter all the fields:

https://t.me/socks?server=84.53.239.95&port=4145&user=myuser&pass=mypassword

Reply:
i added a commit to support the parameters `&user=` and `&pass`